sktime/benchmarking/validations.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+"""Implement and register validations, which compute benchmark results."""
+import functools
+from typing import Callable, List
+
+from sktime.benchmarking import validation_registry
+from sktime.datasets import load_airline
+from sktime.forecasting.base import BaseForecaster
+from sktime.forecasting.model_evaluation import evaluate
+from sktime.forecasting.model_selection import ExpandingWindowSplitter
+from sktime.forecasting.model_selection._split import BaseSplitter
+from sktime.performance_metrics.base import BaseMetric
+from sktime.performance_metrics.forecasting import MeanSquaredPercentageError
+
+
+def forecasting_validation(
+    dataset_loader: Callable,
+    cv_splitter: BaseSplitter,
+    scorers: List[BaseMetric],
+    estimator: BaseForecaster,
+) -> List:
+    """Run validation for forecasting benchmarks."""
+    y = dataset_loader()
+    results = {}
+    for scorer in scorers:
+        scorer_name = scorer.name
+        scores_df = evaluate(forecaster=estimator, y=y, cv=cv_splitter, scoring=scorer)
```

Comment:
Haven't investigated fully but we might want to add a cv_splitter.clone() or cv_splitter.reset() b.c. cv_splitter seems to have state which could get updated by evaluate, though this might not be an issue in practice.